design/workitems.go

```diff
@@ -112,6 +112,7 @@ var _ = a.Resource("workitem", func() {
 			a.Param("filter[iteration]", d.String, "IterationID to filter work items")
 			a.Param("filter[workitemtype]", d.String, "work item type to filter work items by")
 			a.Param("filter[area]", d.String, "AreaID to filter work items")
+			a.Param("filter[workitemstate]", d.String, "work item state to filter work items by")
```

Comment:
I think you should use the name "state" instead of "workitemstate" as we should eventually reduce usage of the term 'workitem' in the productm since it is a product jargon ;)